src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.html

```diff
@@ -28,18 +33,38 @@
                         class="btn btn-primary w-100 synced"
                         [ngClass]="getOverviewStatus()"
                         [ngbTooltip]="'artemisApp.examParticipation.examOverview' | artemisTranslate"
-                        (click)="changePage(true, -1)"
+                        (click)="changePage(true, false, -1)"
                     >
                         <fa-icon [icon]="faBars"></fa-icon>
                     </button>
                 }
             </div>
             <div class="navigation-item">
-                <button class="btn btn-secondary w-100" (click)="changePage(false, exerciseIndex - 1, false)" [disabled]="exerciseIndex <= 0">
+                <button
+                    class="btn btn-secondary w-100"
+                    (click)="changePage(false, quizExam ? exerciseIndex === 0 : false, exerciseIndex - 1, false)"
+                    [disabled]="exerciseIndex + (quizExam ? 1 : 0) <= 0"
+                >
                     <span aria-hidden="true">&laquo;</span>
                     <span class="sr-only">Previous</span>
                 </button>
             </div>
+            <!-- quiz exam item -->
+            @if (quizExam) {
+                <div class="navigation-item">
+                    @if (!examTimeLineView) {
+                        <button
+                            class="btn btn-primary w-100 synced"
+                            [ngClass]="setQuizExamButtonStatus()"
```

Comment:
is this function call avoidable?

Reply:
Yes, it's avoidable. Thanks for catching this. The method call on the template is removed.